pkgs/development/libraries/lib2geom/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, fetchFromGitLab
+, cmake
+, ninja
+, pkg-config
+, boost
+, glib
+, gsl
+, cairo
+, double-conversion
+, gtest
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lib2geom-unstable";
+  version = "2021-01-07";
```

Comment:
Asked upstream to create a tag: https://gitlab.com/inkscape/lib2geom/-/merge_requests/48#note_584705175